inventory-framework-platform/src/main/java/me/devnatan/inventoryframework/pipeline/PlatformOpenInterceptor.java

```diff
@@ -86,15 +85,24 @@ IFRenderContext createRenderContext(IFOpenContext openContext) {
         renderContext.setEndless(openContext.isEndless());
         openContext.getStateValues().forEach(renderContext::initializeState);
 
-        for (final Viewer viewer : openContext.getIndexedViewers().values()) {
-            if (!viewer.isTransitioning()) viewer.setActiveContext(renderContext);
-            // TODO Pass viewer object as parameter instead
-            root.onViewerAdded(renderContext, viewer.getPlatformInstance(), renderContext.getInitialData());
-            root.getPipeline().execute(StandardPipelinePhases.VIEWER_ADDED, renderContext);
-            renderContext.addViewer(viewer);
+        for (final Viewer viewer : renderContext.getIndexedViewers().values()) {
+            setupViewer(viewer, renderContext, root);
         }
 
+        final Viewer viewer = renderContext.getViewer();
+        if (viewer != null) setupViewer(viewer, renderContext, root);
+
```

Comment:
This code opens support to include players in the context of another player without needing to added then through `open(...)`.